USER_MANUAL.MD

```diff
@@ -0,0 +1,5 @@
+# SpatialOS GDK for Unreal Test Gyms Project 
```

Comment:
Could you add a bit about project layout. What things go in which directories etc.